build.py

```diff
@@ -51,6 +51,21 @@ def build_font_instance(generator, instance_descriptor, *steps):
 
     setattr(instance.info, "openTypeOS2Panose", [2, 11, 6, 9, 2, 0, 0, 2, 0, 4])
 
+    instance.info._openTypeGaspRangeRecords =[
```

Comment:
I'm concerned that this property starts with an underscore -- it indicates it might be private. @Madig, are we safe to change this do y'know?

Reply:
I can remove the underscore if it helps :). Apparently it works the same either way.

Reply:
The current version without underscores is correct. We use class properties (getters & setters) for some info attributes to automatically convert lists and dicts into the underlying data structures, the underscore attribute then holds the actual data.

Reply:
(except that ufolib2 (and the UFO spec) had the GASP settings [enumerated wrong](https://github.com/fonttools/ufoLib2/pull/44) and so I bypassed the system entirely) 😄

Reply:
Good catch. My coworker suggested that we always use the flags together so we never noticed.